<launch>
    <!-- sicktoolbox node  -->
    <node name="LIDAR" pkg="sicktoolbox_wrapper" type="sicklms">
        <param name="port" value="/dev/serial/by-id/usb-Prolific_Technology_Inc._USB-Serial_Controller-if00-port0" />
        <param name="baud" value="38400"/>
    </node>

    <node name="encoder" pkg="rosserial_python" type="serial_node.py">
        <param name="port" value="/dev/serial/by-id/usb-Silicon_Labs_CP2104_USB_to_UART_Bridge_Controller_00FEBA77-if00-port0"/>
    </node>

    <!-- <node name="steering" pkg="rosserial_python" type="serial_node.py">
        <param name="port" value="/dev/serial/by-id/usb-Silicon_Labs_CP2104_USB_to_UART_Bridge_Controller_00FEBA3D-if00-port0"/>
    </node> -->

    <node name="imu" type="bno055_arduino_bridge_node" pkg="bno055_arduino_bridge" required="true" output="screen">
        <param name="~serial_port" type="string" value="/dev/serial/by-id/usb-Silicon_Labs_CP2104_USB_to_UART_Bridge_Controller_00FEBA78-if00-port0"/>
        <param name="~serial_baud" type="int" value="115200"/>
    </node>

    <node name="tf_logger" pkg="tf_logger" type="tf_logger" output="screen"/>

    <node pkg="rosbag" type="record" name="record" output="screen" args="-a -o /home/nvidia/bagfiles/gmapping/"/>
</launch>
